pkg/network/ovn_kubernetes.go

```diff
@@ -157,19 +157,19 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["OVNDbServiceType"] = corev1.ServiceTypeClusterIP
 	data.Data["OVNSbDbRouteHost"] = nil
 	data.Data["OVN_SB_NODE_PORT"] = nil
+	data.Data["OVN_NB_DB_ENDPOINT"] = fmt.Sprintf("ssl:%s:%s", bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.OVNSbDbRouteHost, OVN_SB_DB_ROUTE_PORT)
+	data.Data["OVN_SB_DB_ENDPOINT"] = fmt.Sprintf("ssl:%s:%s", bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.OVNSbDbRouteHost, OVN_SB_DB_ROUTE_PORT)
 	pubStrategy := bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.ServicePublishingStrategy
 	if pubStrategy != nil && pubStrategy.Type == hyperv1.Route {
 		if pubStrategy.Route != nil && pubStrategy.Route.Hostname != "" {
 			data.Data["OVNSbDbRouteHost"] = pubStrategy.Route.Hostname
 		}
 	} else if pubStrategy != nil && pubStrategy.Type == hyperv1.NodePort {
 		data.Data["OVNDbServiceType"] = corev1.ServiceTypeNodePort
-		if pubStrategy.NodePort != nil && pubStrategy.NodePort.Port > 0 {
-			data.Data["OVN_SB_NODE_PORT"] = pubStrategy.NodePort.Port
-		}
+		data.Data["OVN_SB_NODE_PORT"] = bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.OVNSbDbRouteNodePort
+		data.Data["OVN_NB_DB_ENDPOINT"] = fmt.Sprintf("ssl:%s:%d", bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.OVNSbDbRouteHost, bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.OVNSbDbRouteNodePort)
```

Comment:
@zshi-redhat the NB_DB_ENDPOINT doesn't seem useful, right? Can we get rid of it?

Reply:
ovnk still uses nbdb endpoint for hybrid case if I remember correctly, we would need to remove that in ovnk, then change cno, although  hybrid won't work today in hypershift in 4.11 due to the same reason.